pkg/logs/input/container/docker_test.go

```diff
@@ -81,7 +81,48 @@ func (suite *DockerTailerTestSuite) TestParseMessage() {
 
 	_, _, _, err = parser.ParseMessage(msg)
 	suite.Equal(errors.New("Can't parse docker message: expected a whitespace after header"), err)
+}
 
+func (suite *DockerTailerTestSuite) TestComputeLogsMetadataFromLabel() {
+	var labels map[string]string
+	var config *config.LogsConfig
+	var valid bool
+
+	config, valid = suite.tailer.extractLogsMetadataFromLabels(labels)
+	suite.Nil(config)
+	suite.False(valid)
+
+	// logs metadata should be null because of wrong format
```

Comment:
A common practice is to do smaller unit tests and name them after what you're testing (just like what you are doing in the comment).